tests/test_service.py

```diff
@@ -1,19 +1,19 @@
 """This modules includes unit tests for the service.py module."""
 
-
 from datetime import datetime
 from six.moves import mock
-
+from pkg_resources import DistributionNotFound
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.


Also, please use alphabetical order inside of the group.